docs/faq/index.rst

```diff
@@ -1,17 +1,276 @@
 .. _faqs:
 
-===
-FAQ
-===
+====
+FAQs
+====
 
-I think I've found a security problem! What should I do?
-   If you think you've found a security problem with Flocker (or any other ClusterHQ software), please send a message to security@clusterhq.com.
-   Your message will be forwarded to the ClusterHQ security team (a small group of trusted developers) for triage and it will not be publicly readable.
+.. contents::
+   :local:
+   :backlinks: none
+   :depth: 2
 
-Due to the sensitive nature of security issues, we ask you not to send a message to one of the public mailing lists.
-ClusterHQ has a policy for :ref:`reporting-security-issues` designed to minimize any damage that could be inflicted through public knowledge of a defect while it is still outstanding.
+Troubleshooting
+---------------
+
+Flocker doesn't seem to working, how can I check the logs?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you think the Flocker services are running correctly, you can check the services or logs using the following methods:
+
+* To check the dataset agent service and logs, run the following commands (the commands are the same for the other services as well):
+
+  On Ubuntu:
+
+  .. prompt:: bash $
+
+     service flocker-dataset-agent status
+     tail /var/log/flocker/flocker-dataset-agent
+
+  On CentOS / RHEL:
+
+  .. prompt:: bash $
+
+     systemctl status flocker-dataset-agent -l
+     journalctl -a -u flocker-dataset-agent
+
+* You can check the Docker container logs:
+
+  .. prompt:: bash $
+
+     docker logs <name or UID of Flocker container>
+
+For more information, see :ref:`flocker-logging`.
+
+My volume says it’s in a different state than I expected, why is this?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are a number of reasons this could happen.
+Your volume could be in detached, pending, attached, or deleted state.
```

Comment:
I think these states are things which are only shown by the unofficial `flockerctl`? How do I see them?